app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+# Helper methods for session handling
+module SessionsHelper
+  def log_in(token)
+    cookies.permanent[:remember_token] = token
+  end
+
+  def log_out
+    cookies.delete(:remember_token)
+    self.current_user = nil
+  end
+
+  def logged_in?
+    !current_user.nil?
+  end
+
+  def digest(token)
+    Digest::SHA1.hexdigest(token)
+  end
+
+  def current_user
```

Comment:
If you are not using this method in views... This should be in the Application controller.

Reply:
It is being used in the view.